src/kernels/composable_kernel/include/tensor_operation/xdlops_gemm.hpp

```diff
@@ -885,16 +911,17 @@ struct XdlopsGemm_t
             auto pa = reinterpret_cast<const data_type*>(&a);
             auto pb = reinterpret_cast<const data_type*>(&b);
 
-            constexpr index_t nxdlops =
-                (sizeof(FloatA) * mfma_type.num_input_blks) / (mfma_type.k * sizeof(data_type));
-
 #if WORKAROUND_SWDEV_229564
```

Comment:
Please move the definition of WORKAROUND_SWDEV_229564 from this file to config.hpp. So all marcros are defined there

Reply:
Not all.

Reply:
The definitions should have as small scope as possible.